interface/super/edit_list.php

```diff
@@ -1238,11 +1311,18 @@ function mysubmit() {
              *   If the edit_options setting of the main list entry is set to 1,
              *    then the list items with edit_options set to 1 will show.
              */
+
+            /**
+             * In case when we are have a get parametr "list_id_container", we are set manually variable list_id
+             * and range from...to if these is set
+             */
+            $sql_limits = "LIMIT ".$list_from.( $list_to >=0 ? ", ".($list_to - $list_from) : "");
```

Comment:
need to do following to prevent sql-injection:
```
$sql_limits = "LIMIT " . escape_limit($list_from) . ( $list_to >=0 ? ", " . escape_limit($list_to - $list_from) : "");
```